src/main/resources/avro/alleleAnnotations.avdl

```diff
@@ -0,0 +1,198 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines types used by the GA4GH Allele Annotation API.
+*/
+
+protocol AlleleAnnotations{
+
+import idl "common.avdl";
+import idl "metadata.avdl";
+import idl "variants.avdl";
+
+/*
+
+The VariantAnnotation record groups different types of annotation records by
+Variant.
+
+The TranscriptEffect sub record holds information on the effect of a specific
+allele on a specific transcript.
+As Variants may overlap multiple transcripts, they may have multiple
+TranscriptEffect records. Variants with multiple alternate alleles will have
+multiple TranscriptEffect records per transcript.
+(2 alternate alleles x 3 transcripts = 6 TranscriptEffect records)
+
+VariantAnnotation records belong to VariantAnnotationSets.
+VariantAnnotationSets are created by comparing a number of Variants from a
+VariantSet to a specific set of reference data using specific software tools.
+A VariantAnnotationSet contains information on reference data and software
+versions used in calculating the annotation; it is essential this information
+is exhaustive.
+
+*/
+
+/**
+An AnalysisResult record holds the output of a prediction package such
+as SIFT on a specific allele.
+*/
+record AnalysisResult {
+
+  /** The ID of the analysis record for this result */
+  string analysisId;
+
+  /** The text-based result for this analysis */
+  union { null, string} result;
+
+  /** The numeric score for this analysis  */
+  union { null, int} score;
+
+}
+
+/**
+An allele location record holds the location of an allele relative to a
+non-genomic coordinate system such as a CDS or protein and holds the
+reference and alternate sequence where appropriate
+*/
+record AlleleLocation {
+
+  /** Relative start position of the allele in this coordinate system */
+  int start ;
+
+  /** Relative end position of the allele in this coordinate system  */
+  union { null, int } end = null;
+
+  /** Reference sequence in feature (this should be the codon at CDS level) */
+  union { null, string } referenceSequence = null;
+
+  /** Alternate sequence in feature (this should be the codon at CDS level) */
+  union { null, string } alternateSequence = null;
+}
+
+/**
+Impact is a simple prioritization for the effect of an allele which is used
+in the annotation record.
+IMPORTANT:
+ Prioritization methods are a crude estimates and are not assumed to be
+ reliable: a 'HIGH' Impact may actually not cause any disruption
+ in protein function or expression.
+*/
+enum Impact {
```

Comment:
We have not yet decided whether to include Impact. The issue is discussed in #516 and will be discussed in the VATT call on March 9th. In the meantime, let's remove impact now to unstick protobuf and compliance work. We can reconsider at a later date.